src/main/java/project/service/UserService.java

```diff
@@ -84,15 +88,49 @@ public User getUser(String userId, String accessTokenFromClient) throws Unauthor
         return user;
     }
 
+
     private void checkIfExistingUserIsAuthorized(final String userId, final String accessTokenFromDatabase,
```

Comment:
everything here below in this method is regarding -> https://github.com/pajdak3/hermes-server/pull/23/files#r41947837

we will start using the authentication like this, but I am very open to rethink it and design a new, better authentication flow